e2e/transform/transform-environment/babel.config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = {
+  overrides: [
```

Comment:
Why `overrides`?

Reply:
A bit new to `@babel/preset-typescript`, I used override as the combination of `@babel/preset-typescript`, `@babel/preset-env` was not working earlier.

On closer analysis the issue was the target of '@babel/preset-env', have made the change